u8timeseries/backtesting/__init__.py

```diff
@@ -1,2 +0,0 @@
-from .backtesting_udf import get_train_val_series, backtest_autoregressive_model
-from .forecasting_simulation import simulate_forecast_ar, simulate_forecast_regr
```

Comment:
Can we still do `from u8timeseries.backtesting import simulate_forecast_ar` without these?

Reply:
No, this does not work anymore. Now it would have to be imported like this: `from u8timeseries.backtesting.forecasting_simulation import simulate_forecast_ar`. Unless there's another way I don't know about.

Reply:
Well I believe that if you don't remove these lines in the sub-modules `__init__.py` files it should still work? In any case, I don't think the users should be exposed to the names of the files we are using - i.e., users should write `from u8timeseries.backtesting import simulate_forecast_ar` but not `from u8timeseries.backtesting.forecasting_simulation import simulate_forecast_ar`

Reply:
Ok yes, I think I misinterpreted your reply on Slack. I will reintroduce the submodule `__init__.py` file imports.